gr-digital/include/gnuradio/digital/clock_recovery_mm_cc.h

```diff
@@ -37,6 +37,18 @@ namespace gr {
      * This implements the Mueller and Müller (M&M) discrete-time
      * error-tracking synchronizer.
      *
+     * The input signal must have values around zero, as it works with
+     * sign changes of its input.
+     *
```

Comment:
"The peak to peak input signal amplitude must be symmetrical about zero, as the M&M timing error detector (TED) is a decision directed TED, and this block uses a symbol decision slicer referenced at zero.

The input signal peak amplitude should be controlled to a consistent level (e.g. +/- 1.0) before this block to achieve consistent results for given gain settings; as the TED's output error signal is directly affected by the input amplitude.